packages/botonic-react/src/constants.js

```diff
@@ -155,4 +155,11 @@ export const ROLES = {
   TRIGGER_BUTTON: 'trigger-button',
   TYPING_INDICATOR: 'typing-indicator',
   TEXT_BOX: 'textbox',
+  WEBVIEW: 'webview',
+  WEBVIEW_HEADER: 'webview_header',
+  STYLED_MESSAGE: 'styled-message',
```

Comment:
If `STYLED_MESSAGE`, `IMAGE_MESSAGE`, `AUDIO_MESSAGE`, `VIDEO_MESSAGE` and `DOCUMENT_MESSAGE` are used to specify the type of message (not the role) maybe they should be defined in a new constant named `MESSAGE_TYPE`.

Reply:
in fact, i suggest not having references of the style: 'styled-whatever', so we should have the role 'message', 'webchat', ...